pom.xml

```diff
@@ -215,16 +219,19 @@
             </plugin>
 
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
I'm against specifying this one (default) `groupId`. It worked without it, didn't it?